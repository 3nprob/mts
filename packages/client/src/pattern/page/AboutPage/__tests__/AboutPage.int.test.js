import { async, regular } from './AboutPage.int.story'

jest.mock('react-router-dom', () => ({ Link: () => <div /> }))

describe('<AboutPage />', () => {
  afterEach(() => jest.clearAllMocks())

  describe('Render', () => {
    test('must match async', () => {
      expect(global.renderToJSON(async())).toMatchSnapshot()
    })

    test('must match regular', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
