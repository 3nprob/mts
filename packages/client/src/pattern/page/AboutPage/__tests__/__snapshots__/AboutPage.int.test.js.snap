// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutPage /> Snaphot must match asyncRender 1`] = `null`;

exports[`<AboutPage /> Snaphot must match defaultRender 1`] = `
<div
  className=""
>
  <div
    className="aboutTemplate"
  >
    AboutTemplate under construction
  </div>
</div>
`;
